verl/utils/dataset.py

```diff
@@ -372,17 +406,18 @@ def __getitem__(self, index):
 
         processed_images = []
         original_dimensions = []  # Store original image dimensions
+        processed_time_series = []
 
         # Extract data_source and dataset
-        vision_path = row_dict['images']
-        if len(vision_path) == 0:
-            vision_path = row_dict['videos']
-        if len(vision_path) == 0:
-            row_dict["data_source"] = "unknown"
-            row_dict["dataset"] = "unknown"
-        vision_path = vision_path[0]
-        row_dict["data_source"] = vision_path.split("/")[0]
-        row_dict["dataset"] = vision_path.split("/")[1]
+        # vision_path = row_dict['images']
+        # if len(vision_path) == 0:
+        #     vision_path = row_dict['videos']
+        # if len(vision_path) == 0:
+        #     row_dict["data_source"] = "unknown"
+        #     row_dict["dataset"] = "unknown"
+        # vision_path = vision_path[0]
+        # row_dict["data_source"] = vision_path.split("/")[0]
+        # row_dict["dataset"] = vision_path.split("/")[1]
```

Comment:
The data_source and dataset key are still needed in validation calculation. Would be great if you can add this back (and maybe add some special processing for time series). The data_source here should be modality (i.e. `ecg`) and the dataset here should be dataset (i.e. `cpsc`).